manual/deps/src/lib.rs

```diff
@@ -1,8 +1,16 @@
+#[cfg(test)]
+mod example;
```

Comment:
The `example` module and its submodules exist to make sure the examples actually work.  Getting them to run directly from the mdbook source would have required awkward reshaping; it also doesn't seem like a good idea to have heavyweight stuff in doctests.

Reply:
sgtm, is there a good place to document that these tests should be updated in tandem with the examples?

Reply:
I've added comments to the example sections in encryption.md, which also has the benefit of documenting while file corresponds to which example :)